### ISSUE TITLE
Description does not mention thius is MySQL only

### ISSUE THREAD

**Issue by mudrd8mz** [PatLemaire/moodle-local_id2db#1]
There is hard-coded support for MySQL only for some reason. I can't really understand why but if there was a reason, it must be stated clearly in the plugin description.

Why not to use the moodle_database layer instead of PDO?

**Comment by PatLemaire** [comment]
A good suggestion !
This is an internal development that I wanted to propose to the community following an intervention in the moodle-moot FR 2014. I did not really have time to adapt it. But I will do it as soon as possible.
In the meantime, I keep the idea of a precision in the description.
Thank you

**Comment by mudrd8mz** [comment]
Yeah, I understand and appreciate you are sharing your custom in-house tools with the wider community. But we also realised that once a plugin is publishe din the plugins directory, folks have quite high expectations and simple assume it will automatically work for them as-is. So we try and encourage the plugins authors to improve their plugins as much as possible before the approval. Thanks for understanding.
